Automattic/jetpack
# Boost: Add Boost version to Tracks events

Append the Jetpack Boost plugin version to the event properties when recording an event from Jetpack Boost.

Fixes #21330 

#### Changes proposed in this Pull Request:

* Append plugin version as an eventProp while recording a Tracks event.
* Note: Plugin version is recorded in both the events recorded from Javascript as well as PHP.

#### Jetpack product discussion


Slack: C0232B8CUHG/p1633501217003100

#### Does this pull request change what data or activity we track or use?


Yes, we have a privacy doc update qued to be changed. Added the same: p93VbO-os-p2

#### Testing instructions:





* Go to a site with Jetpack Boost Plugin activated.
* Navigate to wp-admin/admin.php?page=jetpack-boost.
* Enable `Optimize CSS Loading` module.
* Once the process completes, navigate to the Tracks dashboard and verify the recorded event has eventprops: `boost_version` in ~5 to 7 minutes.
